client/package.json

```
@@ -15,7 +15,7 @@
     },
     "scripts": {},
     "dependencies": {
-        "@types/vscode": "^1.25.0",
+        "@types/vscode": "^1.53.0",
```

```suggestion
        "@types/vscode": "~1.56.0",
```

package-lock.json을 보니 1.91.0이 설치되었네요.
"^" 대신에 "~" 사용해 주세요.

그리고 InputBoxOptions.title은 1.56.0 버전부터 지원되고 있네요.

engines.vscode 속성은 "^1.56.0"으로 수정해 주고요.

---

네, engines.vscode 속성을 포함하여, client/package-lock.json 파일도 npm i 로 1.91 -> 1.56 으로 수정하였습니다.